elixir_phoenix/web/controllers/client_controller.ex

```diff
@@ -0,0 +1,58 @@
+defmodule ElixirPhoenix.ClientController do
+  use ElixirPhoenix.Web, :controller
+
+  alias EboshiApiShootoutElixirPhoenix.Client
+  alias EboshiApiShootoutElixirPhoenix.Repo
+
+  plug :scrub_params, "client" when action in [:create, :update]
+
+  def index(conn, _params) do
+    clients = Repo.all(Client)
+    render(conn, "index.json", clients: clients)
+  end
+
+  def create(conn, %{"client" => client_params}) do
```

Comment:
Of course, this project is more for your exploration than for anything else, so you don't have to listen to any of my feedback if you don't want to, so let me know if you're feeling "JUST SHUT UP AND MERGE IT ITS GREEN". That said, this looks like YAGNI to me. I'm guessing you could delete from line 14 down, and all the tests would still pass.